### PR TITLE
chore: add issue&pr template

### PR DESCRIPTION
**Pull request by xtang**
This pull request intents to help our contributors to open an issue or submit any proposal easier according to the community guidelines

- Add pull request template which incorporates CLA 
- Add issue template:
  - disable blank issue creation
  - bug report
  - feature request
  - enhancement

### CODE REVIEW

**Review by sunng87 — APPROVED** [review]

File: ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  - type: dropdown
+    id: subsystem
+    attributes:
+      label: What subsystems are affected?
+      description: You can pick multiple subsystems.
+      multiple: true
+      options:
+        - Frontend
+        - Datanode
+        - Metaserver
```
Comment (sunng87) [inline ISSUE_TEMPLATE/bug_report.yml:38]:
```suggestion
        - Meta
```

File: ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      required: true
+
+  - type: dropdown
+    id: subsystem
+    attributes:
+      label: What subsystems are affected?
+      description: You can pick multiple subsystems.
+      multiple: true
+      options:
+        - Frontend
```
Comment (sunng87) [inline ISSUE_TEMPLATE/bug_report.yml:36]:
```suggestion
        - Standalone mode
        - Frontend
```

**Review by killme2008 — APPROVED** [review]
LGTM

**Comment by waynexia** [comment]
Looks like we should place these templates under `.github/`, otherwise it won't make efforts

**Pull request merged**